### PR TITLE
Add RSSI, SNR, and datarate to PoC receipt details

### PR DESCRIPTION
**Pull request by danielcolinjames**
preview screenshots:

in summary in context of hotspot:
Screen Shot 2021-07-14 at 2 51 34 PM

in PoC Receipt txn details page:
Screen Shot 2021-07-14 at 2 51 53 PM

### CODE REVIEW

File: components/InfoBox/TxnDetails/BeaconDetailsPane.js [hunk 1]
```diff
@@ -46,9 +46,9 @@ const WitnessesWidget = ({
 }) => {
   const { selectHotspot } = useSelectedHotspot()
   return (
-    <div className={classNames(`bg-gray-200 p-3 rounded-lg col-span-2`)}>
+    <div className={classNames('bg-gray-200 p-3 rounded-lg col-span-2')}>
```
Comment (allenan) [inline components/InfoBox/TxnDetails/BeaconDetailsPane.js]: as long as we're cleaning things up, looks like `classNames` isn't being used here

File: components/InfoBox/TxnDetails/BeaconDetailsPane.js [hunk 2]
```diff
@@ -61,23 +61,49 @@ const WitnessesWidget = ({
...
+                        formatDistance(
+                          calculateDistance(
+                            [challengeeLon, challengeeLat],
+                            [witnessLng, witnessLat],
+                          ),
+                        )}
+                    </span>
+                    <span>Datarate</span>
+                    <span className="text-gray-800 font-medium">
+                      {w.datarate}
```
Comment (allenan) [inline components/InfoBox/TxnDetails/BeaconDetailsPane.js]: maybe wrap with a tooltip to explain? goes for RSSI and SNR too, but I'm not familiar at all with datarate for example

**Review by allenan — APPROVED** [review]
nice addition, just a couple comments otherwise this can merge

File: components/InfoBox/TxnDetails/BeaconDetailsPane.js (hunk 1 above)
Comment (danielcolinjames) [reply components/InfoBox/TxnDetails/BeaconDetailsPane.js]: ha woops yeah good catch

File: components/InfoBox/TxnDetails/BeaconDetailsPane.js (hunk 2 above)
Comment (danielcolinjames) [reply components/InfoBox/TxnDetails/BeaconDetailsPane.js]: good call! done 👍

**Pull request merged**